### DISCUSSION TITLE
v4.0.0

### DISCUSSION THREAD

**Discussion by utarwyn** [utarwyn/discord-tictactoe#330] [Announcements]
## What's Changed

> ⚠️ **Do NOT** use v4 of the module with discord.js v13 or v12! This will **NOT** work.
> ✨ Version 3 is still supported and bugfixes will also be made on it. Check [readme](https://github.com/utarwyn/discord-tictactoe#installation) for more info.

* GH-287: Update to **discord.js v14** by @Gorniaky — https://github.com/utarwyn/discord-tictactoe/pull/276
* GH-287: Move to **Node.js 16.9+**
* GH-292: Add MessageContent intent if using text command — 529b8145bf4f23fcb33fdeccd183d3745949596e
* GH-292: Deprecate usage of text command — 32bba8e258ffe59f2e02edb15266a98ec398443b
* Update Brasilian language — https://github.com/utarwyn/discord-tictactoe/pull/325

**Full Changelog**: https://github.com/utarwyn/discord-tictactoe/compare/v3.1.0...v4.0.0

A huge thanks to @Gorniaky for adapting the module for discord.js v14!

<hr /><em>This discussion was created from the release <a href='https://github.com/utarwyn/discord-tictactoe/releases/tag/v4.0.0'>v4.0.0</a>.</em>

**Comment by Gorniaky** [comment]
🎉
